pkg/agent/clusterclaim.go

```diff
@@ -61,6 +63,16 @@ func (c *agentController) createManagementClusterClaim(ctx context.Context) erro
 	return createOrUpdate(ctx, c.spokeClustersClient, managementClaim)
 }
 
+func (c *agentController) createHostedClusterCountClusterClaim(ctx context.Context, count int) error {
+	if count > (util.MaxHostedClusterCount - 1) {
```

Comment:
```suggestion
	if count >= util.MaxHostedClusterCount {
```
My own personal preference. Please ignore if you think the current way is better.

Reply:
Yes defintely. Done thank you